security/tls-security-profiles.adoc

```diff
@@ -10,7 +10,12 @@ TLS security profiles provide a way for servers to regulate which ciphers a conn
 Cluster administrators can choose which TLS security profile to use for each of the following components:
 
 * the Ingress controller
-* the Kubernetes control plane (Kubernetes API server, Kubernetes controller manager, and Kubernetes scheduler)
+* the control plane
```

Comment:
I *think* Ingress Controller takes an upper-case C.

Reply:
Good call, this was the only place I had it lowered.